package.json

```diff
@@ -108,36 +108,37 @@
   },
   "homepage": "https://buttercup.pw",
   "dependencies": {
-    "opencollective": "^1.0.3"
+    "opencollective": "^1.0.3",
+    "react-intl": "^2.4.0"
```

Comment:
Can we move this to devDependencies? Since it will be compiled to the final package

Reply:
Jep